experimental/Schemes/SpaceGerms.jl

```diff
@@ -235,10 +235,22 @@ end
 ### basic functionality for space germs
 
 ##############################################################################
-# note: issubset, isempty, is_canonically_isomorphic, intersect are 
-#       inherited from Spec
+# note: isempty, ==, intersect are inherited from Spec
 ##############################################################################
 
+function issubset(X::AbsSpaceGerm, Y::AbsSpaceGerm)
+  R = ambient_ring(X)
+  R == ambient_ring(Y) || return false
+  point(X) == point(Y) || return false
+  IY=ideal(OO(X),modulus(quotient_ring(OO(Y))))
```

Comment:
This looks to me as if the input should be restricted to space germs working with an `MPolyQuoLocalizedRing` in the background. Is that correct?

Reply:
Yes, because this is the case which is not covered by the issubset functions from AffineSpec.jl.
Can you suggest, how to properly restrict to this case in the function header?
(And maybe we can discuss on the behaviour of modulus and saturated_ideal in trivial cases at some point outside this PR)